### ISSUE TITLE
translate some of our ebooks from lb.com

### ISSUE THREAD

**Issue by sirkitree** [Lullabot/lb-ebook-poc#8]
* https://www.lullabot.com/white-papers/how-not-over-structure-your-content
* https://www.lullabot.com/white-papers/challenges-big-content-teams-and-how-overcome-them
* https://www.lullabot.com/white-papers/dont-let-your-websites-go-rogue

We'll need to go through these and try to translate them into the new format, determine if images fit our new format, which should be a whole chapter/page/etc.

## steps for adding new e-book
1. copy jponch-sample dir and rename to a short identifiable name
2. open cover.md and change the slug to the dir name, the title to the title of the book, and the text below the front matter to something appropriate for the cover 
3. open index.html and change the title
4. create your chapters in the chapters dir, naming each one with chapter-n.md where n is the number of the chapter
5. each chapter should have a title and image change (or no image), and then the text underneath

**Comment by sirkitree** [comment]
My apologies, while working on this I neglected to open a new branch and pushed some of this work to main. opening a new branch for remainder of pdf conversions.

**Comment by sirkitree** [comment]
https://github.com/Lullabot/lb-ebook-poc/tree/8--translate-ebooks

- The latest book I've converted has some problems with the chapters being out of order on the TOC. 
- It also has 14 chapters which points out a flaw in how I've separated the chapters on the TOC: the first column has 5 and the rest are in the 2nd column so it looks unbalanced. Should probably calculate the chapters and divide by 2 instead of the current logic.

I think I'll address these issues once I get one more book translated.

Also I'll note here that instead of trying to pull the full resolution images (which need resized) in this latest one I've just taken screenshots. Not sure which way we should go with that one... probably should get the full res images instead of screenshots.
